pkg/build/api/v1/types.go

```diff
@@ -620,6 +620,9 @@ type GenericWebHookEvent struct {
 
 	// Git is the git information if the Type is BuildSourceGit
 	Git *GitInfo `json:"git,omitempty"`
+
+	// Env contains additional environment variables you want to pass into a builder container
+	Env []kapi.EnvVar `json:"env,omitempty"`
```

Comment:
On Tue, Apr 26, 2016 at 11:55 AM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/build/api/v1/types.go
> <https://github.com/openshift/origin/pull/8477#discussion_r61112419>:
>
> > @@ -620,6 +620,9 @@ type GenericWebHookEvent struct {
> >
> >  	// Git is the git information if the Type is BuildSourceGit
> >  	Git *GitInfo `json:"git,omitempty"`
> > +
> > +	// Env contains additional environment variables you want to pass into a builder container
> > +	Env []kapi.EnvVar `json:"env,omitempty"`
>
> Do we call this "Env" in any other place?
>
At least for builds, this naming/type convention is used for  the
Custom/Source/Docker BuildStrategy types and the BuildRequest type.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8477/files/fde0d585e7272ad1f545600043154b5afd9a9cd0#r61112419>
>